omero/sysadmins/whatsnew.txt

```diff
@@ -7,7 +7,27 @@ What's new for OMERO 5.3 for sysadmins
   the Ice Python bindings if your package manager does not provide the Ice
   Python packages.
 
-- All OMERO web apps can now be installed using ``pip install``.
+- :doc:`version-requirements` has been updated to reflect other changes in
+  version support for 5.3.0 and tentative plans for 5.4.0.
+  
+- All official OMERO.web apps can now be installed from PyPI. You should
+  reinstall your plugins when you upgrade.
 
-- Windows support discontinued, see :doc:`/sysadmins/windows-migration`.
+- Windows support has been discontinued, see
+  :doc:`/sysadmins/windows-migration`.
 
+- OMERO.web has been decoupled from the server and can now be deployed
+  separately. Support for deployment using Apache has been dropped; if your
+  organization’s policies only allow Apache to be used as the external-facing
+  web-server you should configure Apache to proxy connections to an Nginx
+  instance running on your OMERO server i.e. use Apache as a reverse proxy.
+
+- jquery cache is now disabled by default.
+
+- A new script is available to allow users to migrate annotations from Images
+  to Wells which you may wish to run server-side when upgrading. These
```

Comment:
This is a bit confusing.
When/if the annotations are moved, re-indexing will be triggered. This is not part of the server upgrade
What is part of the upgrade is that if there are existing annotations on wells. They will be re-indexed during the upgrade process i.e. no action required from sysadmin